lever-public.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import requests
+import json
+
+COMPANY_ID = 'innogames'
+COMPANY_NAME = 'InnoGames'
+LEVER_JSON = 'https://api.eu.lever.co/v0/postings/{}?mode=json'.format(
+        COMPANY_ID)
+TEAMS = [
+    'Development',
+    'Quality Assurance',
+    'System Administration',
+    'Office IT',
+    'Analytics',
+    'Data Science'
+]
+
+LEVER_SOURCE_TYPE = 'Job+Board'
+LEVER_SOURCE_DETAIL = 'github_jobs_repo'
+HOMEPAGE_BASE = 'https://www.innogames.com/career'
+TEASER_TEXT='# Open Positions @ [InnoGames]({}?s={})\n\n'.format(
+            HOMEPAGE_BASE, LEVER_SOURCE_DETAIL)
+
+def get_listings():
+    r = requests.get(LEVER_JSON, stream=True)
+    jobs = json.loads(r.text)
+    return(jobs)
+
+
+def sanitize_url(input_str, dedup_char, offset=0, out=''):
+    for i in [' ', '[', ']', '(', ')', '/', '\\', '?', '*', ':']:
+        input_str = input_str.replace(i, '-')
+
+    for c in input_str:
+        if offset > 0 and c == dedup_char and out[offset - 1] == dedup_char:
+            continue
+        else:
+            out += c
+        offset += 1
+
+    return out
+
+
+def get_homepage_url(job):
+    url = HOMEPAGE_BASE
```

Comment:
It's unnecessary to assign the constant here if it's assigned to a different value 2 lines below.